build.gradle

```diff
@@ -168,6 +172,8 @@ task createVersionInfoFile {
 
     outputs.file(versionInfoFile)
 
+    onlyIf {env.JOB_NAME != null }
```

Comment:
What is this change for? With using Github actions a lot changed around env variables set by Jenkins... most of them are no longer set...

Reply:
I don't know, but it's what's preventing the versionInfo file from being built in development.

GitHub says you authored this commit :laughing: so if you don't know either, I'm taking it out.